docs/messageData.md

```diff
@@ -2978,24 +3107,10 @@ With this message the organizer lets the server know to expect a federation_init
             "$comment": "public key of the remote organizer"
```

Comment:
here and in other places you refer to a "remote" organizer, it might be useful to define this once at the beginning of the federation protocol to clarify who is considered "local" and who's considered remote. You might also want to consider renaming this to "inviting" / "joining" , as that might be less ambiguous.

Reply:
I agree that the current wording can at some points be ambiguous, but I fear that if we define it at the start and subsequently remove these adjectives in the later definitions it could be even more ambiguous.

Maybe it would be good to sort of explain at the beginning that the documentation (and protocol definition) is written from the perspective of a specific party (i.e. an organizer Alice) who takes on defined role (i.e. Alice could be the organizer receiving the challenge from the other)?